Update kibana MITRE version to `v14.1` [classic] (#4774)
elastic/security-docs: docs/detections/rules-coverage.asciidoc

[NOTE]
====

> This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK® tactics. Additionally, rules must be mapped to the MITRE ATT&CK® version used by {elastic-sec}: `v14.0`. Elastic prebuilt rules that aren't installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique will not be included.

Author: ```suggestion
This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK® tactics. Additionally, rules must be mapped to the MITRE ATT&CK® version used by {elastic-sec}: `v14.1`. Elastic prebuilt rules that aren't installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique will not be included.
```

> This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK®. The coverage page maps detections to the following https://attack.mitre.org/resources/updates/updates-october-2023[MITRE ATT&CK® version] used by {elastic-sec}: `v14.1`. Elastic prebuilt rules that aren't installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique do not appear on the coverage map.

Author: ```suggestion
This page will only include the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK®. The coverage page maps detections to the following https://attack.mitre.org/resources/updates/updates-october-2023[MITRE ATT&CK® version] used by {elastic-sec}: `v14.1`. Elastic prebuilt rules that aren't installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique do not appear on the coverage map.
```

> This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK®. The coverage page maps detections to the following https://attack.mitre.org/resources/updates/updates-october-2023[MITRE ATT&CK® version] used by {elastic-sec}: `v14.1`. Elastic prebuilt rules that aren't installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique will not appear on the coverage map.

Author: ```suggestion
This page only includes the detection rules you currently have installed, and only rules that are mapped to MITRE ATT&CK®. The coverage page maps detections to the following https://attack.mitre.org/resources/updates/updates-october-2023[MITRE ATT&CK® version] used by {elastic-sec}: `v14.1`. Elastic prebuilt rules that aren't installed and custom rules that are either unmapped or mapped to a deprecated tactic or technique do not appear on the coverage map.
```
